### PR TITLE
Feat: Cart page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
cart page
I have added a cart page in Trendy-T.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (ozairkhan2810) [inline .vscode/settings.json]: can remove this

File: Cart-Page/cart.css [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+/*** Cart badge ***/
+.badge__wrapper {
+  display: inline-block;
+  position: relative;
+}
+.badge--logo {
+  font-size: 70%;
+  font-weight: 900;
+  background-color: red;
```
Comment (ozairkhan2810) [inline Cart-Page/cart.css:158]: can use variable name

File: Cart-Page/cart.css [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+.cart__btn:hover {
+  color: var(--primary-color);
+  background-color: transparent;
+  border: 2px solid var(--white-color);
+}
+/*card*/
+.card-product {
+  margin: 2rem 5rem;
+  display: grid;
+  gap: 20px;
```
Comment (ozairkhan2810) [inline Cart-Page/cart.css:229]: use rem to maintain consistency in units

**Review by ozairkhan2810 — APPROVED** [review]
suggest to use variable name for colors and maintain consistency in units
Rest looks code to merge

File: Cart-Page/cart-Responsive.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      grid-template-columns: 1fr;
+    }
+    .cart__detail{
+      order:1;
+    }
+    .card-product{
+      order:2;
+    }
+  }
+  @media screen and (max-width: 550px) {
```
Comment (JantuDeb) [inline Cart-Page/cart-Responsive.css:17]: Is it really necessary to have one media query at 650px and 550px.

**Review by JantuDeb — APPROVED** [review]
Suggested a changes

File: Cart-Page/cart.css (hunk 2 above)
Comment (SaurabhChawda) [reply Cart-Page/cart.css:158]: This is only for badge background color, Should I convert it into a variable?

**Comment by SaurabhChawda** [comment]
> suggest to use the variable name for colors and maintain consistency in units Rest looks code to merge

ThankYou.

File: Cart-Page/cart-Responsive.css (hunk 4 above)
Comment (SaurabhChawda) [reply Cart-Page/cart-Responsive.css:17]: No, ThankYou I have changed the value.

**Pull request merged**